coopr-docs/docs/source/guide/superadmin/plugins.rst

```diff
@@ -362,18 +403,20 @@ Below is an example directory structure:
 
   $COOPR_HOME/
       provisioner/
-          daemon/
-              plugins/
-                  providers/
-                      my_provider/
-                          my_provider.json
-                          my_provider.rb
-                          [any additional data or lib directories]
-                  automators/
-                      my-automator/
-                          my_provider.json
-                          my_provider.rb
-                          [any additional data or lib directories]
+          lib/
+              provisioner/
```

Comment:
yeah, for now.  plugins dir will be configurable (groundwork laid in this release), and at some point provisioner itself can become a gem.